packages/story-editor/src/components/floatingMenu/layer.js

```diff
@@ -73,6 +77,15 @@ function FloatingMenuLayer() {
     return () => setMoveableMount(null);
   }, [setMoveableMount]);
 
+  // When eyedropper is active, always hide the floating menu, too.
+  useEffect(() => {
```

Comment:
@barklund FYI Also added this here to dismiss the floating menu when eyedropping a color.